### ISSUE TITLE
savevalue - VBSE_OLE_NO_PROP_OR_METHOD - Bart VS the Space Mutants 1.1.vpx

### ISSUE THREAD

**Issue by francisdb** [jsm174/vpx-standalone-scripts#10]
```
2023-06-09 10:57:57.423 ERROR [4342280] [CodeViewer::OnScriptError@1176] Script Error at line 1003 : VBSE_OLE_NO_PROP_OR_METHOD
2023-06-09 10:57:57.423 INFO  [4342280] [CodeViewer::AppendLastErrorTextW@3952] Runtime error: Line: 1003, Character: 4, VBSE_OLE_NO_PROP_OR_METHOD
```

```vbs
sub savehs
    savevalue "Highscore", "hiscore", hisc
End Sub
```

**Comment by jsm174** [comment]
The code for `SaveValue` in VPX standalone is commented out, as it tries to write back to an OLE document. I never ran through the write portion of `POLE`:

https://github.com/vpinball/vpinball/blob/b703b17f76f3b2923d0f38d5992f19413ca17302/pintable.cpp#L722-L779

Anyway, even though it is commented out, it still return `S_OK`, which would means it would not throw this error.

My guess is I don't see the `hisc` actually created via a `Dim` anywhere:

```
grep -i hisc Bart\ VS\ the\ Space\ Mutants\ 1.1.vbs

    savevalue "Highscore", "hiscore", hisc
    If (temp <> "") then hisc = CDbl(temp)
```

Also, if you look that table has multiple `Loadhs` subs.  The first version uses a `HighScore` variable which is `dim`ed. The second version, the code doesn't even make sense:

```
Sub Loadhs
    Dim x
    x = LoadValue(TableName, "HighScore")
    If(x <> "") Then HighScore = CDbl(x) Else HighScore = 10000 End If
End Sub
```

```
sub loadhs
    dim temp
    If (temp <> "") then hisc = CDbl(temp)
    temp = LoadValue("Highscore", "match")
End Sub
```

I think if you changed it to `HighScore` and delete the second `loadhs` you'd be okay.  

FWIW, most tables seem to read/write to `txt` files using the `Scripting.FileSystemObject` which standalone does support.

**Comment by francisdb** [comment]
That code indeed does not make a lot of sense

**Comment by jsm174** [comment]
> That code indeed does not make a lot of sense

Yeh, I'm not sure how the wine vbs engine handles multiple subs with the same name. Not sure if last one or first one wins..

**Comment by jsm174** [comment]
minus the duplicate functions in that script, `SaveValue` and `LoadValue` are now implemented in latest standalone. A new file `VPReg.ini` is saved in the preferences folder.

https://github.com/vpinball/vpinball/blob/7351e8978c3851c46d16ba49eae65d14e9f94370/pintable.cpp#L777-L791
https://github.com/vpinball/vpinball/blob/7351e8978c3851c46d16ba49eae65d14e9f94370/pintable.cpp#L850-L867

**Comment by francisdb** [comment]
table fix at  #43
